src/Packages.ts

```diff
@@ -0,0 +1,66 @@
+import * as fs from 'fs'
+import path from 'path'
+
+import { Descriptor } from './Descriptor'
+import { LsifSymbol } from './LsifSymbol'
+
+export class Packages {
+  constructor(public readonly projectRoot: string) {}
+  private cache: Map<string, LsifSymbol> = new Map()
+  public symbol(filePath: string): LsifSymbol {
+    if (path.normalize(filePath) !== filePath) {
+      throw new Error('BOOM FIX THIS')
+    }
+    // TODO: handle end case.
```

Comment:
Is this `TODO` for the `BOOM` line or something else? (If so, would be good to move it the `TODO:` into the `Error`)